plugins/modules/openssl_csr_info.py

```diff
@@ -50,8 +50,12 @@
         choices: [ auto, cryptography ]
 
 seealso:
-- module: community.crypto.openssl_csr
-- module: community.crypto.openssl_csr_pipe
+    - module: community.crypto.openssl_csr
+    - module: community.crypto.openssl_csr_pipe
+    - ref: community.crypto.openssl_csr_info filter <ansible_collections.community.crypto.openssl_csr_info_filter>
+    # - plugin: community.crypto.openssl_csr_info
+    #   plugin_type: filter
```

Comment:
when will these be safe to uncomment?

Reply:
Once we have semantic markup support... (We've sneaked that one into the semantic markup spec, since supporting this requires support from all docs consumers as well, and the semantic markup spec already has `P(...)` for referencing other plugin types.)